streams/build.gradle

```diff
@@ -17,8 +28,13 @@ repositories {
 group 'com.mirego.trikot'
 
 kotlin {
+    android {
+        publishAllLibraryVariants()
+    }
+
     targets {
         fromPreset(presets.jvm, 'jvm')
+        fromPreset(presets.android, 'android')
```

Comment:
Slightly late, but does it make sense to have both jvm and android as targets? What other jvm-based platform do we want to support beside Android?
Does adding a target forces projets to have one more @actual implementation of expect classes?

Reply:
@martingagnon do you have an opinion on that?